Directory.Build.props

```diff
@@ -291,6 +291,8 @@
     <AndroidNamespaceReplacement Include='rxkotlin' Replacement='RxKotlin' />    
     <AndroidNamespaceReplacement Include='native_test' Replacement='Native_Test' />    
     <AndroidNamespaceReplacement Include='httpflags' Replacement='HttpFlags' />    
+
+    <AndroidNamespaceReplacement Include='org.aomedia.avif.android' Replacement='Xamarin.AOMedia.AVIF' />    
```

Comment:
We probably should make this `Xamarin.AOMedia.AVIF.Android` just in case a future version has the same type name in both the `Android` and "root" namespace.

This can be done by removing the `.android` from `org.aomedia.avif.android` and it will automatically work for this "Android" one and any future added ones.

Reply:
Maybe you are right. I always struggle with `Android` as part of namespace. From my experience there is group of users that does not know how to user fully qualified namespaces and/or `global::` prefix for issues with type resolving in Android apps (caused our root namespace `Android`)